lib/DDG/Spice/Movies/Cast.pm

```diff
@@ -0,0 +1,37 @@
+package DDG::Spice::Movies::Cast;
```

Comment:
Now that I think about it a bit more. I'm wondering if "Cast" is a little misleading. Maybe this should be "Featuring" instead of "Cast"?

I would expect "Cast" to provide a movie's cast...